tools/internal_ci/macos/grpc_basictests.cfg

```diff
@@ -15,7 +15,7 @@
 # Config file for the internal CI (in protobuf text format)
```

Comment:
This whole file is just a copy of `grpc_master.cfg`. Is this supposed to be the PR version instead?